lobby_server.yaml

```diff
@@ -1,15 +1,26 @@
 - version: 1.9.0.0
   host: lobby.triplea-game.org
   port: 3304
-  message: |    
+  message: |
     Welcome to TripleA 1.9 - Got a question, please visit the forums: https://forums.triplea-game.org/
     No talking politics in the lobby! Take your political comments to a private game!
     Lobby Rules: https://forums.triplea-game.org/topic/4
-    
-    All automated hosts have been updated to use the latest stable engine 1.9.0.0.12226. It's recommended that 
-    all players download and install the latest release here: http://triplea-game.org/download/
-    
-    Have an idea on how to make TripleA better? If so please post a reply with 1-3 features/changes to this thread: 
+
+    All automated hosts have been updated to use the latest stable engine 1.9.0.0.12226. It's recommended that
+    all players download and install the latest release here: https://triplea-game.org/download/
+
+    Have an idea on how to make TripleA better? If so please post a reply with 1-3 features/changes to this thread:
     https://forums.triplea-game.org/topic/1032/most-wanted-features-changes
-    
-  error_message:  
+  error_message:
+- version: 1.10.0.0
+  host: lobby.triplea-game.org
```

Comment:
Perhaps lobby-staging.triplea-game.org would be better until we promote a new stable?

Reply:
I'm fine either way for now.

Reply:
I can accept that argument.

I have a bit more trouble with the idea that this wasn't the first thing done in a breaking release. At least it's done now. I assumed the long discussed breaking release was still in the wings.